### PR TITLE
Rename all depending files

### PR DESCRIPTION
**Pull request by michelu89**
This PR renames all occurrences of the Open Manufacturing Platform (OMP), Semantic Data Structuring (SDS) working group and BAMM Aspect Meta Model (BAMM) to Eclipse Semantic Modeling Framework (ESMF) and Semantic Aspect Meta Model (SAMM), respectively.

### CODE REVIEW

File: CONTRIBUTING.md
Comment (atextor) [inline CONTRIBUTING.md:1]: This file still contains references to Open Manufacturing Platform, OMP and/or SDS. Please update.

File: test/models/events-model.ts [hunk 1]
```diff
@@ -12,34 +12,34 @@
...
-@prefix unit: <urn:bamm:io.openmanufacturing:unit:1.0.0#>.
-@prefix bamm-c: <urn:bamm:io.openmanufacturing:characteristic:1.0.0#>.
-@prefix bamm-e: <urn:bamm:io.openmanufacturing:entity:1.0.0#>.
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:1.0.0#>.
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:1.0.0#>.
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:1.0.0#>.
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:1.0.0#>.
 @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>.
-@prefix : <urn:bamm:io.openmanufacturing:1.0.0#>.
+@prefix : <urn:samm:org.eclipse.esmf.samm:1.0.0#>.
```
Comment (atextor) [inline test/models/events-model.ts:20]: The `org.eclipse.esmf.samm` namespace is reserved for the meta model and is _not_ to be used for regular models or test models. Use `org.eclipse.esmf.test` for that instead.

**Review by atextor — CHANGES_REQUESTED** [review]

**Review by atextor — APPROVED** [review]

**Review by atextor — CHANGES_REQUESTED** [review]
contributing.md still needs changes

File: CONTRIBUTING.md
Comment (michelu89) [reply CONTRIBUTING.md:1]: Fixe

**Review by atextor — APPROVED** [review]

**Pull request merged**